index.js

```diff
@@ -54,39 +56,48 @@ async function deployActions({run, functionsDir, timeout, memory}) {
 module.exports = {
   // Execute before build starts.
   onPreBuild: async ({utils, constants, inputs}) => {
-    try {
-      if (!process.env.NIMBELLA_LOGIN_TOKEN) {
+    if (!process.env.NIMBELLA_LOGIN_TOKEN) {
```

Comment:
The logic below checks for cached configs. If they exist, the nimbella token need not exist in the environment - isn't that right? Should the predicate here be `if no token && no cached config`?

Reply:
> If they exist, the nimbella token need not exist in the environment - isn't that right?

You're right.